tests/e2e/features/Libraries.feature

```diff
@@ -135,3 +141,35 @@ Feature: Test roundtrip of the application: Libraries
 
     Then I expect current url is '/libraries/id_1'
     And  I expect 'positive' toast to appear with text 'Library is created.'
+    And  I expect field '[data-cy="library-field-url"]' is 'url_1'
+
+    ####################################################
+    ################## Add library    ##################
+    ####################################################
```

Comment:
```suggestion
    ###################################################
    ################### Add library ###################
    ###################################################
```

Reply:
Nope because text will not be align with other.